subhub/api/webhooks/stripeWebhookPipeline.py

```diff
@@ -0,0 +1,52 @@
+from subhub.api.webhooks.stripeChargeCapturedProcessor import StripeChargeCapturedProcessor
+from subhub.api.webhooks.stripeChargeDisputeClosedProcessor import StripeChargeDisputeClosedProcessor
+from subhub.api.webhooks.stripeChargeDisputeCreatedProcessor import StripeChargeDisputeCreatedProcessor
+from subhub.api.webhooks.stripeChargeDisputeFundsReinstatedProcessor import StripeChargeDisputeFundsReinstatedProcessor
```

Comment:
follow Python file naming conventions already present in the repo: lowercase separated by '_' 1-3 words
this is not java with overly verbose file and class names